Source/Core/Common/GekkoDisassembler.cpp

```diff
@@ -189,12 +189,19 @@ static std::string ldst_offs(u32 val)
   return fmt::format("0x{:04X}", val);
 }
 
-static int SEX12(u32 x)
+static std::string psq_offs(u32 val)
 {
-  if ((x & 0x800) != 0)
-    return static_cast<int>(x | 0xFFFFF000);
+  if (val == 0)
```

Comment:
The `else` statement can be removed and the braces can be removed like this:
```c++
if (val == 0)
  return "0";
if ((val & 0x800) != 0)
  return fmt::format("-0x{:04X}", ((~val) & 0xfff) + 1);
return fmt::format("0x{:04X}", val);
```

Reply:
I removed the braces from the similar function above it too.